porting/debugging/configuration/audio.rst

```diff
@@ -0,0 +1,185 @@
+Audio Configuration 
+===================
+
+Quick Reference
+---------------
+Essential configurations::
+
+    # Edit /etc/pulse/touch.pa
+    load-module module-droid-discover rate=48000 quirks=+unload_call_exit
+    
+    # Load glue module if available
+    .ifexists module-droid-glue-24.so
+    load-module module-droid-glue-24
+    .endif
```

Comment:
Now, we don't intend every port to edit `touch.pa`; in fact, doing so will break when Noble-based images roll out (because it'll contain its own changes). We'll provide a way to specify additional parameters to `module-droid-discover` later.

Also `module-droid-discover` already loads `module-droid-glue-*` when required.

Reply:
Noted. Will correct.